README.md

```diff
@@ -377,8 +377,8 @@ gpg --keyserver pool.sks-keyservers.net --recv-keys B9AE9905FFD7803F25714661B63B
 See the section above on [Verifying Binaries](#verifying-binaries) for details
 on what to do with these keys to verify that a downloaded file is official.
 
-Previous releases of Node.js have been signed with one of the following GPG
-keys:
+Previous releases of Node.js and io.js have been signed with one of the
+following GPG keys:
```

Comment:
We don't support any of the io.js releases anymore anyway. So +1 on just saying `Node.js` and leaving it at that.